Dockerfile.distroless

```diff
@@ -0,0 +1,43 @@
+FROM golang:1.17-buster as builder
+
+WORKDIR /workspace
+RUN apt-get update && apt-get install --no-install-recommends -y git gcc g++ ca-certificates mime-support \
+  && git clone https://github.com/drakkan/sftpgo.git
+ARG TAG
+ARG FEATURES
+ENV GOOS=linux GO111MODULE=on GOPROXY="https://proxy.golang.org,direct" CGOENABLED=0 GOFLAGS="-mod=readonly"
```

Comment:
CGOENABLED=0 is wrong, the correct env var is CGO_ENABLED=0, by disabling CGO support you will disable SQLite support.

Since you set the wrong env var SQLite is built. We have build warnings like this one

```
/usr/bin/ld: /tmp/go-link-2962060537/000014.o: in function `unixDlOpen':
/go/pkg/mod/github.com/mattn/go-sqlite3@v1.14.8/sqlite3-binding.c:40639: warning: Using 'dlopen' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
```

I can confirm that sqlite support is working fine. I think we could have unexpected errors in future.

Reply:
thanks for clarifying